modules/grpc/pubsub/pubsub-dest.cpp

```diff
@@ -57,6 +60,14 @@ DestDriver::~DestDriver()
 bool
 DestDriver::init()
 {
+  if (this->batch_bytes > 10 * 1000 * 1000)
```

Comment:
10 * 1000 *1000 is used two times. It may worth adding a constant for it.

Reply:
Thanks, fixed!